src/AnketaBundle/Resources/views/Hlasovanie/dakujeme.html.twig

```diff
@@ -1,11 +1,11 @@
 {% extends 'AnketaBundle::layout.html.twig' %}
 
 {% block content %}
-{% if app.session.hasFlash('anonymizacia') %}
```

Comment:
Vieme, preco toto nefungovalo na arene, ale v produkcii hej?

Reply:
Pozeral som verziu symfony, a na obidvoch strojoch je 2.3, takze to nou asi nebude.